crates/python/src/lib.rs

```diff
@@ -50,3 +50,39 @@ create_init_submodule! {
 fn qcs_sdk(py: Python<'_>, m: &PyModule) -> PyResult<()> {
     init_submodule("qcs_sdk", py, m)
 }
+
+pub(crate) mod utils {
```

Comment:
Can this be its own file?

Also, `utils` is usually a name of last resort, "things which are useful."

Reply:
made a `py_sync.rs` file